src/Svelte/assets/package.json

```diff
@@ -21,7 +21,7 @@
     },
     "peerDependencies": {
         "@hotwired/stimulus": "^3.0.0",
-        "svelte": "^3.0"
+        "svelte": "^3.0 || ^4.0"
     },
     "devDependencies": {
         "@hotwired/stimulus": "^3.0.0",
```

Comment:
Can you update the `devDependencies` svelte also below? That will, in effect, mean that we'll be testing now with svelte 4... but our CI isn't rich enough (and maybe shouldn't be, as the matrix could get huge) to test multiple versions of the JS.

Reply:
Sorry for the delay!

I can't get the test suite to work with svelte 4 :(
Svelte 4 requires ESM and that would require running jest with the --experimental-vm-modules options.
I followed [the guide from svelte-jester](https://github.com/svelteness/svelte-jester/issues/121#issuecomment-1646119671) and tried a lot of options but can't find a working transformer config for our monorepo + ts setup

Reply:
Any update on this ? I would very much like to use Svelte but not being in v4 is a bit of a blocker.
Any chance to try to use vitest as the testing lib only for the Svelte package ?
The majority of svelte devs use Vite as it is part of the Svelte docs, so I don't think sticking with jest is a good choice for the maintainability

Reply:
That's great! Would you mind pushing your work on a separate PR? Then we could take a look and see what the remaining 25% is.

Reply:
@weaverryan Here you go #1202 😉